pontoon/base/static/js/translate.js

```diff
@@ -1916,7 +1938,8 @@ var Pontoon = (function (my) {
       this.user = {
         email: $('#server').data('email') || '',
         name: $('#server').data('name') || '',
-        localizer: isLocalizer($('#server').data('user-translated-locales')),
+        forceSuggestions: $('.force-suggestions').is('.enabled'),
```

Comment:
That was my idea initially. But it didn't work, because we recreate Pontoon object data every time a new project is loaded. So if forceSuggestions setting has changed, we'd override it with whatever the value on page load was.

Now I refactored a bit to only recreate Pontoon object data that requires that.